Makefile

```diff
@@ -5,6 +5,7 @@ EXTRA_BUILD_ARGS =
 IMAGE_VERSION = $(shell poetry version -s)
 PORT = 8000
 ENV_FILE = .env
+TEST_WORKER_COUNT = 8
```

Comment:
Are spaces after `=` allowed in Makefiles?

Reply:
Yep 🙂